medusa/providers/torrent/html/gimmepeers.py

```diff
@@ -0,0 +1,164 @@
+        }
+
+        response = self.session.post(self.urls['login'], data=login_params)
+        if not response or not response.text:
+            logger.log(u'Unable to connect to provider', logger.WARNING)
+            return False
+
+        if re.search('Username or password incorrect!', response.text):
+            logger.log(u'Invalid username or password. Check your settings', logger.WARNING)
+            return False
+
+        return True
+
+    def search(self, search_strings, age=0, ep_obj=None, **kwargs):  # pylint: disable=too-many-locals
+        """
+        Search a provider and parse the results.
+
+        :param search_strings: A dict with mode (key) and the search value (value)
+        :param age: Not used
+        :param ep_obj: Not used
+        :returns: A list of search results (structure)
+        """
+        results = []
+        if not self.login():
+            return results
+
+        for mode in search_strings:
+
+            logger.log(u'Search Mode: {0}'.format(mode), logger.DEBUG)
+            for search_string in search_strings[mode]:
+                if mode != 'RSS':
+                    logger.log(u'Search string: {0}'.format
+                               (search_string.decode('utf-8')), logger.DEBUG)
+
+                self.search_params['search'] = search_string
+
+                data = self.session.get(self.urls['search'], params=self.search_params)
+                if not data:
+                    continue
+
+                results += self.parse(data.text, mode)
+
+        return results
+
+    def parse(self, data, mode):
+        """
+        Parse search results for items.
+
+        :param data: The raw response from a search
+        :param mode: The current mode used to search, e.g. RSS
+        :return: A list of items found
+        """
+        items = []
+
+        with BS4Parser(data, 'html5lib') as html:
+            torrent_table = html.find('table', class_='browsetable')
+            torrent_rows = torrent_table('tr') if torrent_table else []
+
+            # Continue only if one Release is found
+            if len(torrent_rows) < 2:
+                logger.log(u'Data returned from provider does not contain any torrents', logger.DEBUG)
+                return items
+
+            for result in torrent_rows[1:]:
+                try:
+                    cells = result('td')
+
+                    link = cells[1].find('a')
+                    download_url = self.urls['download'] % cells[2].find('a')['href']
+
+                    title = link.getText()
+                    seeders = int(cells[10].getText().replace(',', ''))
+                    leechers = int(cells[11].getText().replace(',', ''))
+                    torrent_size = cells[8].getText()
+                    size = convert_size(torrent_size) or -1
+                    # except (AttributeError, TypeError):
+                    #     continue
+
+                    if not all([title, download_url]):
```

Comment:
This needs to go right after title and download_url. See https://github.com/pymedusa/Medusa/blob/master/medusa/providers/torrent/html/alpharatio.py#L135-L139